src/ts/components/listselector.ts

```diff
@@ -10,11 +10,45 @@ export interface ListItem {
   label: string;
 }
 
+/**
+ * Filter function that can be used to filter out list items added through {@link ListSelector.addItem}.
+ *
+ * Using this does not make a lot of sense when items are added from outside because the added items can be
+ * controlled anyway, but it does make sense to be used in conjunction with subclasses that populate
+ * themselves automatically via the player API, e.g. {@link SubtitleSelectBox}.
+ */
+export interface ListItemFilter {
```

Comment:
Why don't we add a more generic interface. Like: 
```
export interface Predicate<T> {
(t: T): boolean;
}
```

Reply:
Because we don't need it anywhere else. It can be added later if desired, it's just an interface.